src/main/resources/swagger/cromiam.yaml

```diff
@@ -43,8 +59,10 @@ paths:
         '500':
           description: Internal Error
       security:
-        - google_oauth:
+        - googleoauth:
```

Comment:
The `oauth` section is a copy/paste from sam. Since I was pasting whole sections, I just left it as-is